dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp

```diff
@@ -553,4 +563,16 @@ ModelOmegaRef::setFequations() {
   assert(fEquationIndex_.size() == (unsigned int) sizeF() && "ModelOmegaRef:fEquationIndex_.size() != f_.size()");
 }
 
+void
+ModelOmegaRef::checkDataCoherence(const double& /*t*/) {
+  for (int i = 0; i < nbMaxCC; ++i) {
+    if (doubleEquals(yLocal_[i], omegaRef0_[i]))
+      continue;
+    if (yLocal_[i] < omegaRefMin_)
```

Comment:
&& doubleNotEquals to respect dynawo precision

Reply:
Done.